app/src/test/java/com/duckduckgo/app/pixels/EnqueuedPixelWorkerTest.kt

```diff
@@ -73,22 +80,24 @@ class EnqueuedPixelWorkerTest {
     @Test
     fun whenOnStartAndAppLaunchThenSendAppLaunchPixel() {
         whenever(unsentForgetAllPixelStore.pendingPixelCountClearData).thenReturn(1)
+        whenever(webViewVersionProvider.get()).thenReturn("91")
 
         enqueuedPixelWorker.onStateChanged(lifecycleOwner, Lifecycle.Event.ON_CREATE)
         enqueuedPixelWorker.onStateChanged(lifecycleOwner, Lifecycle.Event.ON_START)
 
-        verify(pixel).fire(AppPixelName.APP_LAUNCH)
+        verify(pixel).fire(AppPixelName.APP_LAUNCH, mapOf("webview_version" to "91"))
     }
 
     @Test
     fun whenOnStartAndLaunchByFireActionFollowedByAppLaunchThenSendOneAppLaunchPixel() {
         whenever(unsentForgetAllPixelStore.pendingPixelCountClearData).thenReturn(1)
         whenever(unsentForgetAllPixelStore.lastClearTimestamp).thenReturn(System.currentTimeMillis())
+        whenever(webViewVersionProvider.get()).thenReturn("91")
 
         enqueuedPixelWorker.onStateChanged(lifecycleOwner, Lifecycle.Event.ON_CREATE)
         enqueuedPixelWorker.onStateChanged(lifecycleOwner, Lifecycle.Event.ON_START)
         enqueuedPixelWorker.onStateChanged(lifecycleOwner, Lifecycle.Event.ON_START)
 
-        verify(pixel).fire(AppPixelName.APP_LAUNCH)
+        verify(pixel).fire(AppPixelName.APP_LAUNCH, mapOf("webview_version" to "91"))
```

Comment:
any reason why you prefer using a string instead of `PixelParameter.WEBVIEW_VERSION` in the test?

Reply:
It's mostly for readability and knowing exactly what to expect. If I had done this through TDD (disclaimer: i didn't), I would know that we wanted to have a paramter "webview_version" before I would know where and how to define it in the implementation.